sdk/appconfiguration/app-configuration/CHANGELOG.md

```diff
@@ -9,14 +9,16 @@
 
 ### Breaking Changes
 
+- Support for Node.js 8 and IE 11 has been dropped. Please see our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
+- Changed TS compilation target to ES2017 in order to produce smaller bundles and use more native platform features
+- Updated our internal core package dependencies to their latest versions in order to add support for Opentelemetry 1.0.0 which is compatible with the latest versions of our other client libraries.
```

Comment:
Can we move these under Features?
If we have something under `Breaking Changes` then the expectation is that there is a major version change

Reply:
Moved